src/CliMenu.php

```diff
@@ -235,6 +235,18 @@ protected function executeCurrentItem()
     }
 
     /**
+     * Redraw the menu
+     */
+    public function reDraw()
```

Comment:
sorry to be pedantic but redraw is one word right? it just looks a little odd as `reDraw` what do you think ?